html-aam/index.html

```diff
@@ -9365,6 +9365,132 @@ <h4 id="att-controls">`controls`</h4>
             </tr>
           </tbody>
         </table>
+        <h4 id="att-command">`command`</h4>
+        <table aria-labelledby="att-command">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>`command`</td>
+            </tr>
+            <tr>
+              <th>Element(s)</th>
+              <td><a data-cite="html/popover.html#attr-command">`button`</a>;</td>
+            </tr>
+            <tr>
+              <th>[[WAI-ARIA-1.2]]</th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th>
+                <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
+              </th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th>Comments</th>
+              <td>The value of this attribute can impact the mappings of its related <a href="#att-commandfor">`commandfor`</a> attribute.</td>
```

Comment:
so the rational (flawed or not) popovertargetaction was listed as not having mappings, and instead adjusted the popovertarget attribute's mappings is because popovertarget was the constant for the button that invoked the popover.

with command/commandfor though, it doesn't seem like there would be a default action that would occur without a command value specified, and in such a case this attribute would have no additional mappings for the button?

where i'm going with this, is that if this were just going to ever have commands for popover/dialogs, then it'd probably make sense to stick with how this is spec'd now.

but since this has the potential to expand into different commands, where each of those commands might then need to update the mappings further, maybe it would end up being easier to spec if we treated this similar to how `input type` elements are spec'd, where each value (or collection of values?) for `type` is given its own mapping table.

what lead me to think about this is that the current commandfor mappings don't mention the "show-modal" value, and how all the mappings that are there for the popover related values need to go away for that one.


would this change in how the spec is written have an impact on how this has been implemented?   if so, then maybe there's just a different formatting to the spec that could be done to make this easier to maintain (and i can handle that in another PR down the line)